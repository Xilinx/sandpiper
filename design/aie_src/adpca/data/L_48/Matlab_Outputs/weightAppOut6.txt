6.631567001e+00 -2.207035542e+00
-7.032079697e-01 -2.595072031e+00
1.146160603e+00 -6.349925995e+00
