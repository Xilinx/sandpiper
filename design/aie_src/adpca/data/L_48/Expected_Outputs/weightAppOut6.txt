6.631372452e+00 -2.207030296e+00 
-7.030645609e-01 -2.594957829e+00 
1.146044254e+00 -6.349828720e+00 
